horovod/mxnet/tensor_util.cc

```diff
@@ -135,11 +135,9 @@ void TensorUtil::Free(NDArray* tensor) { delete tensor; }
 
 // Resize tensor to nDimension with length size[i] in dimension i
 void TensorUtil::ResizeNd(NDArray* tensor, int nDimension, int64_t* size) {
-  TShape mx_shape(nDimension);
-  for (int idx = 0; idx < nDimension; ++idx) {
-    mx_shape[idx] = size[idx];
-  }
-  tensor->Reshape(mx_shape);
+  void* temp_out;
```

Comment:
nit: this is not really a temp variable. renaming it to ndarray_handle for better readability?